pos_product_available_negative/static/src/js/pos.js

```diff
@@ -9,6 +10,25 @@ odoo.define('pos_product_available_negative.pos', function (require) {
     var core = require('web.core');
     var _t = core._t;
 
+    models.PosModel = models.PosModel.extend({
+        check_qty_of_orderlines: function(orderlines, product, quantity_to_add) {
```

Comment:
* Function name mismatches with what it does
* You can take `orderlines` from inside of the func. Get rid of the argument. Don't forget to remove lines where `orderline` variable was defined in the code for the func

Reply:
check